ONSdigital/eq-author-app
# fix for long section names

### What is the context of this PR?

Section names are pushing the error dot out of the container when too long

### How to review 

1) Create a new section with a name (shortcode) longer than the container in the left hand section nav container.
2) make sure that section has an error - eg no section title
3) the section name (shortcode) should revert to 3 dots near the end and NOT push the error dot out of the container.
4) should line up with any error dots in the question section of the nav
5) all above should hold true when resizing browser - (within reason!)

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
